src/graphics/raster.rs

```diff
@@ -95,15 +94,28 @@ mod librsvg_imports {
             mock! {
                 pub Loader {
                     pub fn new() -> Self;
-                    pub fn read_stream<S: glib::IsA<gio::InputStream>, F: glib::IsA<gio::File>, P: glib::IsA<gio::Cancellable>>(&self, stream: &S, base_file: Option<&'static F>, cancellable: Option<&'static P>) -> Result<SvgHandle, LoadingError>;
+                    pub fn read_stream<S, F, P>(
+                        &self,
+                        stream: &S,
+                        base_file: Option<&'static F>,
+                        cancellable: Option<&'static P>
+                    ) -> Result<SvgHandle, LoadingError>
+                    where
+                        S: glib::IsA<gio::InputStream>,
+                        F: glib::IsA<gio::File>,
+                        P: glib::IsA<gio::Cancellable>;
```

Comment:
This, and the `render_document` function down below, have been formatted manually. This wasn't necessary/strictly related to updating the dependencies, but I've been meaning to do it for a while because they were running off the screen.

Note that these have to be formatted manually because they are contained within a macro call (both `mock!` and `cfg_if!`, denoted by the trailing `!`), and our automated code formatting tool (rustfmt) doesn't really support formatting args to macro calls. That's because rustfmt knows/deals with formatting valid syntax, but macros are able to define their own inputs which often isn't strictly valid Rust code outside a macro call context.

As such this block will have to be formatted by hand, though the formatting changes made here are "correct" in the sense that's how rustfmt would format them outside a macro call